### PR TITLE
Removing event log file

### PR DESCRIPTION
**Pull request by dimpar**
Ref: keep-network/keep-ecdsa#574

This event log that is used in `keep-ecdsa` will be replaced by generated function from keep-commons.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -168,6 +168,8 @@ github.com/karalabe/usb v0.0.0-20191104083709-911d15fe12a9 h1:ZHuwnjpP8LsVsUYqTq
 github.com/karalabe/usb v0.0.0-20191104083709-911d15fe12a9/go.mod h1:Od972xHfMJowv7NGVDiWVxk2zxnWgjLlJzE+F4F7AGU=
 github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4 h1:CivupPSFswHACua5xZGKdeYxsCQ2cmRomTIBh8kfk70=
 github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
+github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce h1:B+kei2KjTdLS7JeK6YeDFU6suzPbkCn8WiAX1VDqtA0=
+github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce/go.mod h1:emxogTbBdey7M3jOzfxZOdfn139kN2mI2b2wA6AHKKo=
```
Comment (pdyraga) [inline go.sum:172]: Let's `go mod tidy`.

Comment (dimpar) [reply go.sum:170]: Done.

File: go.mod [hunk 2]
```diff
@@ -5,5 +5,5 @@ go 1.13
 require (
 	github.com/ethereum/go-ethereum v1.9.10
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4
+	github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce
```
Comment (pdyraga) [inline go.mod:8]: Can we update it to `8af057907255bf852d9b2dca15e9e3d1af4ecc02`? This is https://github.com/keep-network/keep-common/pull/58 merged to `master`.

Comment (dimpar) [reply go.mod]: Done.

**Review by pdyraga — APPROVED** [review]

**Comment by pdyraga** [comment]
Ooops @dimpar, unsigned commits.

**Pull request merged**